### PR TITLE
Mention extras and examples

### PR DESCRIPTION
**Pull request by rizar**
Addresses #830 (does not solve this issue, because examples need more comments). 

Please let me know where else we should refer to blocks-examples.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -23,6 +24,7 @@ Blocks is developed in parallel with Fuel_, a dataset processing framework.
 .. _mailing list: https://groups.google.com/forum/#!forum/blocks-users
 .. _making a GitHub issue: https://github.com/mila-udem/blocks/issues/new
 .. _Fuel: https://github.com/mila-udem/fuel
+.. _examples: https://github.com/mila-udem/blocks-examples
```
Comment (dmitriy-serdyuk) [inline docs/index.rst]: You don't need to define this reference again, Shynx doesn't like it. https://github.com/mila-udem/fuel/pull/232#discussion_r39973585

**Comment by rizar** [comment]
@nke001, can you please check if referring to Blocks-examples this way solves the documentation issue that you came across yesterday?

File: docs/index.rst (hunk 1 above)
Comment (rizar) [inline docs/index.rst]: But how I do two different link texts then?

**Comment by nke001** [comment]
Hi, thanks for modifying the doc. It looks a lot more clear to me this way.

**Comment by rizar** [comment]
@dmitriy-serdyuk  is away for a few days, a new reviewer is needed. @lamblin , could you please help?

**Comment by rizar** [comment]
Thanks for reviewing it quickly!

**Pull request merged**